profiles/foot.lua

```diff
@@ -160,6 +160,9 @@ function way_function (way, result)
   if "roundabout" == junction then
     result.roundabout = true
   end
+  if "circular" == junction then
+    result.circular = false
```

Comment:
But we do for rotaries right? Above we set the roundabout flag, which means large names roundabouts will be announced as rotaries already.